README.md

```diff
@@ -1,3 +1,9 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+### How to initialize this project
+
+This repository uses the npm dotenv package to store some environment variables, to run this project you need to create two files `.env.test` and `.env.development` in the root folder of this project, and inside each file it should contain the following environment variable `PGDATABASE=` with the name of your database.
+
+Example:
+
+`PGDATABASE=name_of_your_database`
```

Comment:
The setup.sql file contains specific names of nc_news_test & nc_news , the readme should reflect that as the setup.sql file is not git ignored so users are likely to utilise the same names